packages/components-js/tests/e2e/puppeteer/specs/validate-versions.e2e.ts

```diff
@@ -0,0 +1,66 @@
+import { Page } from 'puppeteer';
+import {
+  enableBrowserLogging,
+  getConsoleWarnings,
+  getOldLoaderScriptForPrefixes,
+  initConsoleObserver,
+  setContentWithDesignSystem,
+} from '../helpers';
+import { version } from '@porsche-design-system/components-js/package.json';
+import type { PorscheDesignSystem } from '@porsche-design-system/components/dist/types/bundle';
+
+let page: Page;
+beforeEach(async () => {
+  page = await browser.newPage();
+  initConsoleObserver(page);
+});
+afterEach(async () => await page.close());
+
+it('should show warning about multiple different versions correctly', async () => {
+  enableBrowserLogging(page);
+
+  const prefixes = ['test', 'my-prefix'];
+
+  await setContentWithDesignSystem(
+    page,
+    `<p-text>Some Text</p-text>
+    ${getOldLoaderScriptForPrefixes(prefixes)}`,
+    {
+      withoutWaitForComponentsReady: true,
+    }
+  );
+
+  const porscheDesignSystem = await page.evaluate(() => document.porscheDesignSystem as PorscheDesignSystem);
+
+  expect(porscheDesignSystem[version]).toBeDefined();
+  expect(porscheDesignSystem['3.7.0']).toBeDefined();
+  expect(porscheDesignSystem['3.7.0'].prefixes).toEqual(prefixes);
+
+  const versionWarning = getConsoleWarnings().find((warning) => warning.text().includes('Multiple different versions'));
+
+  expect(versionWarning).toBeDefined();
+
+  const warningArgs = await Promise.all(versionWarning.args().map(async (arg) => await arg.evaluate((arg) => arg)));
```

Comment:
Let's use `getConsoleWarnings()` for consistency.

Reply:
I already use getConsoleWarnings but i need the args separately because if i just convert the ConsoleMessage into text i can't check the object argument properly.